mappings/markdown/Zeek/conn_log/README.md

```diff
@@ -1,55 +1,73 @@
 # Event Dossier: Zeek conn.log
-### 
+### Summary:
 - **Description**: Translates a Zeek conn.log to OCSF. 
 
 - **Event References**:
+  - https://schema.ocsf.io/1.3.0/classes/network_activity
   - https://docs.zeek.org/en/master/logs/conn.html
-  - https://schema.ocsf.io/1.0.0-rc.3/classes/network_activity
+  - https://docs.zeek.org/en/master/scripts/base/protocols/conn/main.zeek.html#type-Conn::Info
  
- ### OCSF Version: 1.0.0-rc.3
- - `category_uid`: `4`
+ ### Static mapping: OCSF Version 1.3.0
+ - `metadata.version`: `1.3.0`
  - `category_name`: `Network Activity`
- - `class_uid`: `4001`
+ - `category_uid`: `4`
  - `class_name`: `Network Activity`
- - `metadata.profiles`: `[security_control]`
- - `metadata.product.name`: `Zeek`
+ - `class_uid`: `4001`
+ - `metadata.log_name`: `conn.log`
+ - `metadata.loggers[].log_provider`: `Zeek`
+ - `metadata.loggers[].product.name`: `Zeek`
+ - `metadata.product.cpe_name`: `cpe:2.3:a:zeek:zeek`
  - `metadata.product.feature.name`: `conn.log`
+ - `metadata.product.name`: `Zeek`
+ - `metadata.product.url_string`: `https://docs.zeek.org/en/current/logs/conn.html`
  - `metadata.product.vendor_name`: `Zeek`
  - `severity`: `Informational`
  - `severity_id`: `1`
 
  ### Mapping:
- - This does not reflect any transformations or evaluations of the data. Some data evaluation and transformation will be necessary for a correct representation in OCSF that matches all requirements.
-
- - Any fields not present in an explicit mapping will be mapped to the unmapped object. 
 
 | OCSF                          | Raw           |
 | ----------------------------- | --------------|
-|'time'                         |'ts'           |
-|'uuid'                         |'uid'          |
-|'src_endpoint.port'            |'id.orig_p'    |
-|'dst_endpoint.ip'              |'id.resp_h'    |
-|'dst_endpoint.port'            |'id.resp_p'    |
-|'connection_info.protocol_name'|'proto'        |
-|'bytes_in'                     |'orig_bytes'   |
-|'packets_in'                   |'orig_pkts'    |
-|'orig_bytes.ip'                |'orig_ip_bytes'|
-|'bytes_out'                    |'resp_bytes'   |
-|'packets_out'                  |'resp_pkts'    |
-|'resp_bytes.ip'                |'resp_ip_bytes'|
-|'duration'                     |'duration'     |
-|'unmapped'                     |'conn_state'   |
-
- ### Conditional Mapping:
- - Any fields described within the conditional mappings are subject to dynamic mappings contingent on a conditional evaluation of source data. Fields which fail to meet a particular conditional are assigned a default value from the OCSF schema description.
-| OCSF                 | Raw             | Notes                       |
-| ---------------------| ----------------| ----------------------------|
+|`time`                         |`ts`           |
+|`uuid`                         |`uid`          |
```

Comment:
I don't think `uuid` is right here. Rather, I think this goes into `metadata.uid`.

Reply:
Thank you - good find.  I updated this.